hbase-client/src/main/java/org/apache/hadoop/hbase/client/RawAsyncHBaseAdmin.java

```diff
@@ -1185,41 +1184,41 @@ private void checkAndGetTableName(byte[] encodeRegionName, AtomicReference<Table
   }
 
   @Override
-  public CompletableFuture<Void> mergeRegions(byte[] nameOfRegionA, byte[] nameOfRegionB,
-      boolean forcible) {
+  public CompletableFuture<Void> mergeRegions(List<byte[]> nameOfRegionsToMerge, boolean forcible) {
+    if (nameOfRegionsToMerge.size() < 2) {
```

Comment:
just to confirm my understanding, we're doing this ourselves instead of using Preconditions because guava doesn't have a version of Preconditions that will return a future?

Reply:
I would think so too. Preconditions doesn't seem to have the capability to return a future object that we would need for these functions.